src/neuroconv/datainterfaces/behavior/deeplabcut/deeplabcutdatainterface.py

```diff
@@ -0,0 +1,92 @@
+"""Heberto Mayorquin, Authors: Saksham Sharda, Cody Baker, Ben Dichter."""
+from typing import Optional
+from pathlib import Path
+
+from pynwb.file import NWBFile
+
+from ....basedatainterface import BaseDataInterface
+from neuroconv.utils import dict_deep_update
+from neuroconv.utils import FilePathType, OptionalFilePathType
+from neuroconv.tools.nwb_helpers import make_or_load_nwbfile
+
+try:
+    from dlc2nwb.utils import auxiliaryfunctions, write_subject_to_nwb
+
+    HAVE_DLC2NWB = True
+except ImportError:
+    HAVE_DLC2NWB = False
+
+
+class DeepLabCutInterface(BaseDataInterface):
+    """Data interface for DeepLabCut datasets"""
+
+    def __init__(
+        self,
+        file_path: FilePathType,
+        config_file_path: FilePathType,
+        subject_name: str = "ind1",
+        verbose: bool = True,
+    ):
+        """
+        Interface for writing DLC's h5 files to nwb using dlc2nwb.
+        Parameters
+        ----------
+        file_path: FilePathType
+            path to the h5 file output by dlc.
+        config_file_path: FilePathType
+            path to .yml config file
+        subject_name: str
+            the name of the subject for which the nwbfile is to be created.
+        verbose: bool
+            controls verbosiy. True by default.
```

Comment:
```suggestion
        verbose: bool = False,
    ):
        """
        Interface for writing DLC's h5 files to nwb using dlc2nwb.
        Parameters
        ----------
        file_path: FilePathType
            path to the h5 file output by dlc.
        config_file_path: FilePathType
            path to .yml config file
        subject_name: str
            the name of the subject for which the nwbfile is to be created.
        verbose: bool
            controls verbosity. False by default.
```

Reply:
I am of the opinion that all the verbose should be False by default but @CodyCBakerPhD is of a different opinion. In favor of his side of the argument, at the point that this discussion surfaced all the other interfaces had this default.

Reply:
I see. I prefer default False as well, but let's stay consistent for now